operator/src/main/java/oracle/kubernetes/weblogic/domain/model/Cluster.java

```diff
@@ -88,6 +88,15 @@
   @Range(minimum = 0)
   private Integer maxConcurrentStartup;
 
+  @Description(
+          "The maximum WebLogic Server instances that will shutdown in parallel "
```

Comment:
"The maximum number of WebLogic". "shutdown" -> "shut down". "count. " (add space). Add a comma after "`spec.maxClusterConcurrentShutdown`" and then change "(which in turn defaults to 1)." to "which defaults to 1."